packages/widgetbook_cli/bin/parsers/exceptions.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:widgetbook_exception/widgetbook_exception.dart';
+
+class WidgetbookDeployFailure extends WidgetbookException {
+  WidgetbookDeployFailure(super.message);
```

Comment:
All these exceptions should have a pre-defined message. Otherwise, we could also make `WidgetbookException` non abstract and remove this code.